mods/ra/rules/aircraft.yaml

```diff
@@ -340,43 +340,46 @@ HIND:
 	Inherits@AUTOTARGET: ^AutoTargetGroundAssaultMove
 	Buildable:
 		Queue: Aircraft
-		BuildAtProductionType: Helicopter
-		BuildPaletteOrder: 20
-		Prerequisites: ~hpad, ~techlevel.medium
-		Description: Helicopter gunship armed\nwith dual chainguns.\n  Strong vs Infantry, Light armor\n  Weak vs Tanks, Aircraft
+		BuildAtProductionType: Plane
+        BuildPaletteOrder: 35
+		Prerequisites: ~afld, ~techlevel.medium
+		Description: Fast infantry transport helicopter\n Armed with dual chainguns\n Strong vs Infantry.\n Weak vs Tanks, Aircraft
 	Valued:
-		Cost: 1500
+		Cost: 1000
 	Tooltip:
 		Name: Hind
 	UpdatesPlayerStatistics:
 		AddToArmyValue: true
 	Health:
 		HP: 10000
 	RevealsShroud:
-		Range: 10c0
+		Range: 9c0
 		Type: GroundPosition
 		RevealGeneratedShroud: False
 	RevealsShroud@GAPGEN:
-		Range: 8c0
+		Range: 7c0
 		Type: GroundPosition
 	Armament@PRIMARY:
-		Weapon: ChainGun
+		Weapon: PChainGun
 		LocalOffset: 85,-213,-85, 85,213,-85
 		MuzzleSequence: muzzle
-		PauseOnCondition: !ammo
 	Armament@SECONDARY:
 		Name: secondary
-		Weapon: ChainGun
+		Weapon: PChainGun
 		LocalOffset: 85,213,-85, 85,-213,-85
 		MuzzleSequence: muzzle
-		PauseOnCondition: !ammo
 	AttackAircraft:
 		FacingTolerance: 20
 		PersistentTargeting: false
 	Aircraft:
-		LandWhenIdle: false
 		TurnSpeed: 4
 		Speed: 112
+		LandableTerrainTypes: Clear,Rough,Road,Ore,Beach,Gems
+		AltitudeVelocity: 0c58
+    Cargo:
+        Types: Infantry
+        MaxWeight: 4
+        PipCount: 4
```

Comment:
More spaces instead of tabs.

Reply:
It seems that some of the copy-pasted text had spaces or that they were created when pasting